__tests__/plugin.test.ts

```diff
@@ -1439,6 +1457,16 @@ describe('babel plugin', () => {
       expect(code).not.toContain('version: ');
       expect(code).toMatchSnapshot();
     });
+
+    it('uses `runWithTaggedExceptions` for worklets', () => {
```

Comment:
Not sure if test name should contain implementation details like the function name
```suggestion
    it('throws a tagged exception when processing worklet fails', () => {
```